package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "_s",
+  "version": "1.0.0",
+  "scripts": {
+    "css-build": "npm run css-lint && npm run css-compile && npm run css-prefix && npm run css-format && npm run css-fix",
+    "css-lint": "stylelint style.css",
+    "css-compile": "node-sass sass/style.scss style.css --output-style expanded --indent-type tab --indent-width 1",
+    "css-prefix": "postcss style.css -u autoprefixer -o style.css --no-map",
+    "css-format": "stylefmt style.css",
+    "css-fix": "stylelint style.css --fix",
+    "scss-lint": "stylelint sass/**/*.scss",
+    "js-lint": "eslint js/**",
+    "js-fix": "eslint js/** --fix"
+  },
+  "devDependencies": {
+    "autoprefixer": "^7.1.2",
+    "browserslist": "^2.1.5",
+    "eslint": "^4.1.1",
+    "eslint-config-wordpress": "^2.0.0",
+    "node-sass": "^4.5.3",
+    "postcss-cli": "^4.1.0",
+    "stylefmt": "^6.0.0",
+    "stylelint": "^7.12.0",
+    "stylelint-config-wordpress": "^11.0.0"
+  },
+  "stylelint": {
+    "defaultSeverity": "warning",
+    "extends": "stylelint-config-wordpress",
+    "rules": {
+      "max-line-length": [ 80, {
+        "ignorePattern": ["/\b(Description:).*\n/", "/https?:\/\/[0-9,a-z]*.*/"]
+        } ]
+      }
+    },
+    "eslintConfig": {
+      "extends": "wordpress"
+    },
+    "eslintIgnore": [
+      "js/**.min.js"
+    ],
+    "browserslist": [
+      "> 1%",
+      "last 2 versions",
+      "IE 11"
+    ]
```

Comment:
The browserlist matrix should match that of WordPress' https://make.wordpress.org/design/handbook/design-guide/browser-support/

You can see the above in code form [https://core.trac.wordpress.org/changeset/41062/trunk/Gruntfile.js here]